app/konnect/org-management/users.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Manage Users
+no_version: true
+---
+
+You can invite users to join your Konnect organization through the **Organization**
+page.
+
+To manage user access, see [Manage Teams and Roles](/konnect/org-management/teams-and-roles).
+
+{:.note}
+> **Note:** If Okta integration is [enabled](/konnect/org-management/okta-idp),
+{{site.konnect_short_name}} users and teams become read-only. An organization
+admin can view all registered users in {{site.konnect_short_name}}, but cannot
+edit their team membership from the {{site.konnect_short_name}} side. To manage
+automatically-created users, adjust user permissions through Okta, or
+[adjust team mapping](/konnect/org-management/okta-idp/#map-teams-to-groups).
+
+## Prerequisites
+
+You must be part of the **Organization Admin** team to manage users.
+
+## Add a user to the organization
+
+### Invite a user
+1. In {{site.konnect_saas}}, open the ![](/assets/images/icons/konnect/icn-organizations.svg){:.inline .no-image-expand .konnect-icon}
+ **Organization > Users** page.
+1. Select **Invite User**.
+1. Enter the user’s name and email.
+1. Assign the user to one or more teams.
+
+    For team descriptions, hover over the information (`i`) icon next to the team name,
+    or see the [predefined teams references](/konnect/org-management/teams-reference).
+
+1. Click **Save**.
+
+    An email invitation is sent to the user.
+
+### Accept invite and create account
+
+1. From the invitation email, follow the link to set up your account.
+1. Create a password.
+
+    The password must be 8 characters long and contain at least three of the
```

Comment:
This feels like it could become out of date quickly. Is the UI clear enough that we could remove the password requirements here?

Reply:
Nice catch. The UI has been greatly improved since I originally wrote this a year ago:
![Screen Shot 2022-03-29 at 8 59 00 AM](https://user-images.githubusercontent.com/54370747/160654811-68918ecf-7def-4318-90f9-d9214a3a321f.png)

I'll remove it.